### ISSUE TITLE
Play effects sound effects

### ISSUE THREAD

**Issue by tacotexmex** [pyrollo/late#7]
To increase immersion sounds could be played during
- raise,
- still,
- fall,
- end
of an effect. The syntax can be as following:
```
sounds = {
	on_raise = { sound = "raise.ogg", gain = 0.5, length = "1.4", loop = false },
	on_still = { sound = "sound.ogg", gain = 0.5, length = "1.4", loop = true  },
	on_fall = { sound = "sound.ogg", gain = 0.5, length = "1.4", loop = false  },
	on_end = { sound = "sound.ogg", gain = 0.5, length = "1.4", loop = false  },
	to_target = false,
},

```
I think it's pretty self-explanatory. 
- The length var is needed for proper looping of the sound if `loop = true`. 
- `to_target = true` will play the sound to the target (player) only, while when `false` it plays it on target position to everyone.

**Comment by pyrollo** [comment]
I wonder if this can be achieved simply by registering an impact.

**Comment by pyrollo** [comment]
I had many thought about that. Actually, I'm thinking about rewriting the phase system. I'd like to create an "intensity" registry into which I can register "time" and "distance" intensity. This will allow to have also other intensities (item use/wear...).

Raise, still, fall, end phases may be replaced by something else like "start", "terminated", "max intensity reached"... So I'm wondering exactly which sound event would be usefull.

I can imagine a sound playing while effect acting. Another sound when effect starts and/or ends. But I'm not sure at all for other moments.

**Comment by tacotexmex** [comment]
> I can imagine a sound playing while effect acting. Another sound when effect starts and/or ends. But I'm not sure at all for other moments.

Yes, this is what I'm into. I must say I really like the raise, still, fall, end terminology because it's easy to understand. (It's a bit like attack, decay, sustain, release in synthesizers)

**Comment by pyrollo** [comment]
I think, there will be something like : registered intensity combined gives the overall wanted intensity. Then the real intensity will follow it with a raise and fall speed. So you'll keep the raise/still/fall thing but it will be much more flexible.
